### PR TITLE
[13]Style Product Page

### PR DESCRIPTION
**Pull request by Fiyin985**
Screenshot (30)
Screenshot (29)

### CODE REVIEW

**Comment by ToluAkin** [comment]
@Fiyin985 Remove all the bullets on the page. The bullets are from list items

**Comment by Fiyin985** [comment]
Ok

File: submissions/fiyingabriel/HTML/new 1.html [hunk 1]
```diff
@@ -0,0 +1,25 @@
+<!DOCTYPE html>
```
Comment (ToluAkin) [inline submissions/fiyingabriel/HTML/new 1.html]: @Fiyin985 Remove this file

File: submissions/fiyingabriel/css/Product.css [hunk 2]
```diff
@@ -0,0 +1,331 @@
+  
+html {
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:2]: `html` display is `block` by default. remove this.

File: submissions/fiyingabriel/css/Product.css [hunk 3]
```diff
@@ -0,0 +1,331 @@
...
+    margin-left: 40px;
+    height: 450px;
+}
+.detail {
+    margin-top: 57px;
+}
+#qt {
+    text-align: center;
+    width: 45px;
+    padding: 3px 3px 3px 3px;
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:52]: use `padding: 3px` instead.

File: submissions/fiyingabriel/css/Product.css [hunk 4]
```diff
@@ -0,0 +1,331 @@
...
+    border-radius: 1px;
+}
+#buy-now {
+    border-radius: 15px;
+    margin-left: 0px;
+    padding: 10px 10px 10px 10px;
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:58]: Use `padding: 10px` instead

File: submissions/fiyingabriel/css/Product.css [hunk 5]
```diff
@@ -0,0 +1,331 @@
...
+    font-size: 17px;
+}
+/* top display product description */
+.prod-detail {
+    float: left;
+}
+#des {
+    padding-bottom: 1px;
+    font-family: 'Calibri';
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:67]: You've set the font family already. You can add this font to the body's font-family. You should remove it from here.

**Comment by Fiyin985** [comment]
Screenshot (31)
Screenshot (32)
Ive done it

File: submissions/fiyingabriel/css/Product.css [hunk 6]
```diff
@@ -0,0 +1,331 @@
...
+    margin-left: 95px;
+}
+.two, .three, .four {
+    display: inline-block;
+    text-align: center;
+    font-size: 20px;
+    margin-left: 40px;
+    background-color: white;
+    font-weight: bold;
+    padding: 0px 25px 0px 25px;
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:95]: This is same as `padding: 0 25px`. You don't need to add a unit to a `0` value in css.

File: submissions/fiyingabriel/css/Product.css [hunk 7]
```diff
@@ -0,0 +1,331 @@
...
+    margin: 15px 40px 0px 0px;
+}
+/* footer newsletter signUp */
+#newsLetter {
+    margin: 20px 0px 0px 430px; 
+}
+#input-email {
+    width: 185px;
+    height: 15px;
+    padding: 7px 0px 7px 0px;
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:130]: This is same as `padding: 7px 0;`

File: submissions/fiyingabriel/css/Product.css [hunk 8]
```diff
@@ -0,0 +1,331 @@
...
+    margin: 40px 0px 0px 350px;
+    color: white
+}
+
+/* media-query (tablet)*/
+@media screen and (min-width: 768px) and (max-width: 900px) {
+        /* product page */
+    /* top displayed product and detail */
+    .prod-list {
+        margin: 0px 20px 0px 20px ;
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:162]: This is same as `margin: 0 20px;`

File: submissions/fiyingabriel/css/Product.css [hunk 9]
```diff
@@ -0,0 +1,331 @@
...
+        height: 515px;
+    }
+    /* top display product description */
+    #des {
+        padding-bottom: 5px;
+        font-family: 'Calibri';
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:168]: Don't set font family here. Remove this.

File: submissions/fiyingabriel/css/Product.css [hunk 10]
```diff
@@ -0,0 +1,331 @@
...
+    .three {
+        margin-right: 16px;
+    }
+    .three, .four {
+        text-align: center;
+        font-size: 20px;
+        margin-left: 20px;
+        background-color: #004b18;
+        font-weight: bold;
+        padding-right: 0px 30px 0px 30px;
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:189]: `padding-right` should only take one value.

File: submissions/fiyingabriel/css/Product.css [hunk 11]
```diff
@@ -0,0 +1,331 @@
...
+    }
+    .top-prod #pi {
+        float: right;
+        width: 595px;
+        height: 615px;
+        margin-top: 0px;
+        margin-left: 115px;
+    }
+    .top-prod {
+        padding: 0px;
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:238]: Change this to `padding: 0`

File: submissions/fiyingabriel/css/Product.css [hunk 12]
```diff
@@ -0,0 +1,331 @@
...
+    }
+    #des {
+        padding-bottom: 5px;
+        margin-bottom: 0px;
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/Product.css:242]: As mentioned above.

File: submissions/fiyingabriel/css/new 1.css [hunk 13]
```diff
@@ -0,0 +1,12 @@
+#side{
```
Comment (ToluAkin) [inline submissions/fiyingabriel/css/new 1.css]: Remove this file

**Comment by Fiyin985** [comment]
Ive changed it

**Pull request merged**